cmd/client-dump.go

```diff
@@ -251,6 +255,26 @@ func (c *Client) copyOpenFiles(dir string) error {
 	return nil
 }
 
+func (c *Client) checkFileLocks(pid int32) bool {
+	for _, fd := range c.Process.OpenFds {
+		info, err := c.fs.ReadFile(fmt.Sprintf("/proc/%s/fdinfo/%s", strconv.Itoa(int(pid)), strconv.FormatUint(fd.Fd, 10)))
```

Comment:
Can we use /proc/locks for a better algo?

Reply:
just ended up removing the function for now, wasn't really using it anyway